### DISCUSSION TITLE
Get duration in 'mm':'ss' format

### DISCUSSION THREAD

**Discussion by marcobaradel** [goldfire/howler.js#1569] [Q&A]
Hi,
i have a question (probably is a stupid question). I'm using this lib to do an audio player on my app.
I want to realize a box with play and stop button, a progress bar and a realtime counter minutes value on the total duration (ex: 1:24/3:02).
For this last label i'm trying to use the .duration() method, but it return a string with duration in seconds. Exist a method that return it in 'mm':'ss' format or there is a trick to convert it?

I use the variable on the xxxxx.page.html with '{{xxxx}}' format.

I'm waiting for reply,
Thanks for the future help.

**Comment by ymzEmre** [comment]
Hi,

If i didn't misunderstand you, This function can solve the problem.

```
const msToTime = (getTime) => {
  let getMs = new Date(getTime * 1000);
  let getHours = getMs.getHours();
  let getMinutes = getMs.getMinutes();
  let hours = getHours > 9 ? getHours : '0' + getHours;
  let minutes = getMinutes > 9 ? getMinutes : '0' + getMinutes;
  return hours + ':' + minutes;
};
```
